sale_order_line_hls/models/product.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    """ Override the method that compute a multiline description of this product, in the context of sales.
+        In order to display/output product description(sale.order.line.name)  without reference code.
+    """
+    _inherit = 'product.product'
+
+    def get_product_multiline_description_sale(self):
+        return self.name
```

Comment:
Let's not disregard `description_sale` when it's available.  The code should be:
```.py
    def get_product_multiline_description_sale(self):
        name = self.name
        if self.description_sale:
            name += '\n' + self.description_sale
        return name
```